### ISSUE TITLE
Error with packages at running after installing plugin

### ISSUE THREAD

**Issue by ahrherrera** [arnesson/cordova-plugin-firebase#876]
**Describe the bug**
After installing the plugin, there are errors at building, related to packages: Failed to apply plugin [id 'com.google.gms.google-services'], and I fixed it by adding the classpath to the .gradle files, then there is a error Failed to apply plugin [id 'io.fabric']

**To Reproduce**
1. Install plugin
2. Build Ionic Project
3. It will appear build errors targeting to /platforms/android/cordova-plugin-firebase

**Expected behavior**
Build successful and let me use Firebase services

**Screenshots**

**Plugin Version**
"^2.0.2"
**Smartphone (please complete the following information):**
 - Device: Pixel XL
 - OS: Android Oreo
 - Browser WebView
 

**Additional context**

**Comment by soumak77** [comment]
@ahrherrera could you test PR #872 to see if it fixes this issue for you.  That'll help get the fix released faster.  You can check our [contributing doc](https://github.com/arnesson/cordova-plugin-firebase/blob/master/CONTRIBUTING.md#need-to-test-a-pr) for info on how to test a PR.

**Comment by soumak77** [comment]
@ahrherrera I believe this has been fixed via PR #883.  Please verify by installing the plugin from master
```
cordova plugin add https://github.com/arnesson/cordova-plugin-firebase.git#master
```

If you are still encountering the issue, please reopen it.
